src/PostgREST/Types.hs

```diff
@@ -163,7 +163,13 @@ operators = M.fromList [
   ("is", "IS"),
   ("@@", "@@"),
   ("@>", "@>"),
-  ("<@", "<@")]
+  ("<@", "<@"),
+  ("&&", "&&"),
```

Comment:
the old mode can be supported since you can have 2 tuples resolving to the same operator.
my preference would be to be consistent :), all chars, i think `@` was a mistake.

other opinions?